server/models.py

```diff
@@ -196,6 +196,14 @@ class Election(BaseModel):
     )
     standardized_contests = Column(JSON)
 
+    # During audit setup, we compute sample size options for the first
+    # round of the audit in a background task and store the results here.
+    sample_size_options_task_id = Column(
+        String(200), ForeignKey("background_task.id", ondelete="cascade")
```

Comment:
cascade? Shouldn't it be set to null?

Reply:
whoops, just copied it from elsewhere without thinking it through. fixed here: 1746f087. I think there may be other places this is misconfigured as well